content/pull-requests/collaborating-with-pull-requests/addressing-merge-conflicts/resolving-a-merge-conflict-on-github.md

```diff
@@ -59,6 +59,11 @@ If a site administrator disables the merge conflict editor for pull requests bet
    Click **Create branch and update my pull request** or **I understand, continue updating _BRANCH_**. The button text corresponds to the action you are performing.
 1. To merge your pull request, click **Merge pull request**. For more information about other pull request merge options, see "[Merging a pull request](/articles/merging-a-pull-request/)."
 
+ If the conflict is in binary data such as an image or sound file, this method of resolving conflicts cannot be used. 
+ In order to resolve conflicts involving binary files, immidiately make a back up, delete and re-clone the project, then re-introduce non-conflicting files.
+ 
+ For the conflicting files, examine the latest data in the freshly cloned project and decide if you can append your edits in a DCC program or whether to overwrite the changes entirely in a fresh pull request. 
```

Comment:
I don't know what a DCC program is. If referring to this you will need to explain what it is, and possibly link to more information elsewhere.